fog/sample-paykit/proto/remote_wallet.proto

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2018-2021 The MobileCoin Foundation
+
+/// This file contains a simple service wrapper for a wallet that is used by the fog conformance tests
+/// to perform balance checks against an arbitrary client implementation.
+/// The sample-paykit includes such implementation, and the idea is that other clients/SDKs could
+/// also implement this service and then get tested by the fog conformance tests Python script.
+
+syntax = "proto3";
+
+package remote_wallet;
+
+option java_package = "com.example.myapplication.proto";
+option java_outer_classname = "RemoteWalletProto";
+
+import "google/protobuf/empty.proto";
+
+service RemoteWalletApi {
+    /// Start a new client and perform a fresh balance check.
+    rpc FreshBalanceCheck (FreshBalanceCheckRequest) returns (BalanceCheckResponse);
+
+    /// Perform a balance check on a previously-started client.
+    rpc FollowupBalanceCheck (FollowupBalanceCheckRequest) returns (BalanceCheckResponse);
+
+    /// Stop a previously-started client.
+    rpc Stop (StopRequest) returns (google.protobuf.Empty);
+
+    /// Get debug info for a previously-started client.
+    rpc Debug (DebugRequest) returns (DebugResponse);
+
+    /// Drop all previously-started clients.
+    rpc Reset (google.protobuf.Empty) returns (google.protobuf.Empty);
+}
+
+message FreshBalanceCheckRequest {
+    /// Root entropy for the account the new client will be using.
+    bytes root_entropy = 1;
```

Comment:
This was written before this change :) I agree it should change to use bip39_entropy once @jcape's work is merged (since this relies on `keyfile`)

Reply:
In the interim, should we use `view_private_key` and `spend_private_key` instead of `root_entropy`? That would at least mean that it's usable with old clients that still use root_entropy key derivation and new clients that are already using mnemonic key derivation (like the ios sdk) and no longer support root_entropy.

Reply:
The issue with that is that the fog conformance test python script (the only thing that currently uses this remote wallet thing) does not have a way to do the derivation. It would need to rely on some external utility to do so. Whats in `master` is also currently using root entropy. Unless you feel strongly about this, my preference would be to leave it as is in this PR and fix it in a follow-up PR once the `keyfile` changes land.